lib/foreman/renderer.rb

```diff
@@ -26,6 +26,18 @@ def template_logger
       @template_logger ||= Foreman::Logging.logger('templates')
     end
 
+    def host_info(*path)
+      @info ||= @host.info
+      info = @info
+      path.each { |step| info = info.fetch step }
+      info
+    end
+
+    def host_param(*path)
```

Comment:
and to add lazy cache of values, since the the enc takes time and users don't know that they can keep it in some local variable